draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -576,14 +572,9 @@ defined:
   in (#display-metadata). This property is OPTIONAL.
 * `claims`: An array of objects containing claim information for the type, as described in
   (#claim-metadata). This property is OPTIONAL.
-* `schema`
-  * OPTIONAL. An embedded JSON Schema document describing the structure of
-  the Verifiable Credential as described in (#schema-definition). `schema` MUST NOT be used
-  if `schema_uri` is present.
-* `schema_uri`
-  * OPTIONAL. A URL pointing to a JSON Schema document describing the structure
-  of the Verifiable Credential as described in (#schema-definition). `schema_uri` MUST NOT
-  be used if `schema` is present.
+
+A Type Metadata document MAY contain additional top level or subordinate properties.
```

Comment:
You might still want to have an equivalent of extensions that must to be understood just like [the crit header parameter does in JWS](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.11).

Reply:
My intuition is that something like that isn't needed for metadata. I'm not aware of a similar construct in any of the metadata content in the OAuth/OIDC realm nor that occasion that it'd have been useful has come to light.